corehq/apps/geospatial/routing_solvers/pulp.py

```diff
@@ -107,7 +106,7 @@ def calculate_distance_matrix(self):
             'sources': sources,
             'destinations': destinations,
             'annotations': 'distance',
-            'access_token': settings.MAPBOX_ACCESS_TOKEN
+            'access_token': config.plaintext_api_token,
```

Comment:
Will this impact the projects using radial algorithm? they don't need to have a token right? so we should use the one in settings for them?

Reply:
> Will this impact the projects using radial algorithm?
The `RoadNetworkSolver` class inherits from `RadialDistanceSolver`, but the `RadialDistanceSolver` class has it's own `calculate_distance_matrix` method which does not make use of mapbox.